examples/TheraLang.DataSeeding/Program.cs

```diff
@@ -12,28 +13,36 @@ private static void Main(string[] args)
         {
             using var dbContext = CreateDbContext();
 
-            dbContext.ClearAndSeed(Projects());
+            dbContext.ClearAndSeed(ProjectTypes());
+            dbContext.ClearAndSeed(Projects(dbContext.GetArrayOf<ProjectType>()));
```

Comment:
please remove the dbContext.GetArrayOf method, using it adds an additional call to DB - which is unnecessary as you can just get types from ProjectTypes() method - but you will need to add explicit Ids there and enable IDENTITY_INSERT option in db, refer to:
https://docs.microsoft.com/en-us/ef/core/saving/explicit-values-generated-properties#saving-an-explicit-value-during-add